### DISCUSSION TITLE
proto imports

### DISCUSSION THREAD

**Discussion by anaxaim** [AnWeber/vscode-httpyac#155] [Q&A] [answered]
I have a proto file with this import:
`
import "google/api/annotations.proto";
`

I run this grpc code from directory _/api/proto/cards/_
`proto < cards.proto
GRPC /cards.CardsService/GetCardsList
Authorization: {{authToken}}
{
   "clientId": "{{clientId}}"
}`

and get an error:
`Error: ENOENT: no such file or directory, open '.../api/proto/cards/google/api/annotations.proto'`

How can I point to the annotations.proto file so that the import works?

**Comment by AnWeber** [comment, marked as answer]
I can't recreate the behavior. I have made up the following example.

- card.http
```http
proto < cards.proto
GRPC grpcb.in:9000/hello.HelloService/sayHello
{
  "greeting": "world"
}
```
- card.proto
```proto
import "test/hello.proto";
```
- test/hello.proto
```proto
syntax = "proto2";
package hello;
service HelloService {
  rpc SayHello(HelloRequest) returns (HelloResponse);
}
message HelloRequest {
  optional string greeting = 1;
}
message HelloResponse {
  required string reply = 1;
}
```
[prototest.zip](https://github.com/AnWeber/vscode-httpyac/files/9692493/prototest.zip)


Is the annotations.proto present in the right place? Is the relative path to it correct? Can you possibly adjust my example so that it doesn't work. Maybe you must [provide `includeDirs` to proto import](https://github.com/grpc/grpc-node/tree/master/packages/proto-loader#usage).
